frontend/src/components/App.js

```diff
@@ -114,7 +114,8 @@ const App = ({userID, user, logOut, fetchInitialData, fetchAuthorizedData}) => {
 
   useEffect(onDidMount, []);
 
-  useUserInputExpiration(logOut, 10 * 1000); // 1000ms = 1 s
+  // Logout the user after 12 hours in all cases where the tab stays open
+  useUserInputExpiration(logOut, 43200 * 1000); // total 12 hours (1000ms = 1 s)
```

Comment:
I'd create a file `time.js` with
```js
export const second = 1000;
export const minute = second * 60;
export const hour = minute * 60;
export const day = hour * 24; // Slightly inaccurate
```
Then you can simply write `12 * hour`, simple & readable, no need for the comment.